lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -179,6 +179,12 @@ LogicalResult calyx::verifyCell(Operation *op) {
 
 LogicalResult calyx::verifyControlLikeOp(Operation *op) {
   auto parent = op->getParentOp();
+
+  if (!isa<calyx::CalyxDialect>(parent->getDialect())) {
```

Comment:
Your comment says you relaxed the calyx enable verify, but this is relaxing all control-like operation verifiers?

Reply:
Correct; that might be a bit too relaxed, will update.